ManageRDSessions/ManageRDSessions.psm1

```diff
@@ -122,7 +152,7 @@ function Remove-sbRDSession {
                 if ($AsJob.IsPresent) {
                     Write-Verbose "Attempting Logoff of [$($session.Username)] on [$($session.HostServer)] [AsJob]"
                     # Need to use "using:" scope here to pass local hashtable to Job function, otherwise will pass all as null
-                    $sb = {Invoke-RDUserLogoff @using:params}
+                    $sb = { Invoke-RDUserLogoff @using:params }
```

Comment:
Good man. The scoping issues always burn me, before I remember to use `$using:MyVarName`

Reply:
Yeah that was a “fun” one to troubleshoot. I think I’ll need to refer to this a few times in future!